app/Resources/FOSUserBundle/serializer/Model.User.yml

```diff
@@ -0,0 +1,11 @@
+FOS\UserBundle\Model\User:
```

Comment:
@umpirsky IIRC problem is that you __can't__ expose fields in serializer mapping that are declared on parent class, so i.e. username comes from FOSUser not from our Core etc.